### DISCUSSION TITLE
Adding multiple currency conversions

### DISCUSSION THREAD

**Discussion by felixjouminowa** [birchill/10ten-ja-reader#2126] [Ideas]
The currency conversion feature is very very handy, however in many cases I'd use different conversions (ex. JPY-USD, JPY-EUR, USD-EUR) and would be awesome to have multiple conversions displaying option at the same time.

**Comment by birtles** [comment]
Hi! Thanks for the comment. I think it wouldn't be hard to do multiple JPY-(other) conversions, e.g. JPY-USD + JPY-EUR. Normally we detect currencies by looking for 円 and ￥ characters but we recently changed it to also support "yen" as well so you can look up "10,000 yen" and "JPY 2,000" too.

In order to support USD-EUR I guess we'd have to make it detect $ and maybe "USD" etc. too. To do it really properly we'd have to detect all sorts of currency symbols like €. I think we could do it but it would be quite a bit of work.

**Comment by felixjouminowa** [comment]
I'm not a programmer so I have no idea how much work it would require, but
having even just the JPY-USD & JPY-EUR (plus any other personalized
currencies) would be very helpful for sure!

〒389-2502
長野県下高井郡野沢温泉村豊郷6974－1
Lodge Schanze Nozawa
Manager 箕輪 フェリックス 盛
Email: ***@***.***
TEL: 090-4745-2124
